AirCasting/SearchAndFollow/CompleteScreen/SearchCompleteScreenMapView.swift

```diff
@@ -0,0 +1,53 @@
+// Created by Lunar on 25/02/2022.
+//
+
+import SwiftUI
+import UIKit
+import GoogleMaps
+
+struct SearchCompleteScreenMapView: UIViewRepresentable {
+    typealias UIViewType = GMSMapView
+    var longitude: CLLocationDegrees
+    var latitude: CLLocationDegrees
+    
+    init(longitude: CLLocationDegrees, latitude: CLLocationDegrees) {
+        GMSServices.provideAPIKey(GOOGLE_MAP_KEY)
+        self.longitude = longitude
+        self.latitude = latitude
+    }
+    
+    func makeUIView(context: Context) -> GMSMapView {
+        // Zoom:
+        // 1. World
+        // 5. Continent
+        // 10. Cities
+        // 15. Streets
+        // 20. Buildings
+        
+        let camera = GMSCameraPosition.camera(withLatitude: latitude, longitude: longitude, zoom: 15.0)
+        let mapView = GMSMapView.map(withFrame: .zero, camera: camera)
+        
+        do {
+            if let styleURL = Bundle.main.url(forResource: "style", withExtension: "json") {
```

Comment:
this code is mixing low level stuff with the view layer. It's not terrible and can stay IMO, but just pointing it out. Proper design would be to create a `StyleStore` protocol which provides us with styles, hide this `Bundle` related logic inside the implementation and inject it here using dependency injection.